diff --git a/Engine/Config/BaseDeviceProfiles.ini b/Engine/Config/BaseDeviceProfiles.ini
index 24e70afb8675..3181f812dcdb 100644
--- a/Engine/Config/BaseDeviceProfiles.ini
+++ b/Engine/Config/BaseDeviceProfiles.ini
@@ -140,7 +140,8 @@
 +DeviceProfileNameAndTypes=Android_Xclipse_xxx,Android
 +DeviceProfileNameAndTypes=Android_Xclipse_xxx_Vulkan,Android
 +DeviceProfileNameAndTypes=Android_Vulkan_SM5,Android
-+DeviceProfileNameAndTypes=Android_Adreno_Vulkan_SM5,Android
++DeviceProfileNameAndTypes=Android_Adreno7xx_Vulkan_SM5,Android
++DeviceProfileNameAndTypes=Android_Adreno8xx_Vulkan_SM5,Android
 +DeviceProfileNameAndTypes=Android_Mali_Vulkan_SM5,Android
 +DeviceProfileNameAndTypes=Android_Xclipse_Vulkan_SM5,Android
 +DeviceProfileNameAndTypes=Android_PowerVR_G6xxx,Android
@@ -787,7 +788,8 @@ BaseProfileName=VisionPro
 +MatchProfile=(Profile="Android_Adreno5xx_Low",Match=((SourceType=SRC_GpuFamily,CompareType=CMP_Regex,MatchString="Adreno \\(TM\\) 5[0-1][0-9]")))
 +MatchProfile=(Profile="Android_Adreno5xx",Match=((SourceType=SRC_GpuFamily,CompareType=CMP_Regex,MatchString="Adreno \\(TM\\) 5[0-9][0-9]")))
 ; Enable SM5 on Adreno 7xx and 8xx when SM5 is enabled
-+MatchProfile=(Profile="Android_Adreno_Vulkan_SM5",Match=((SourceType=SRC_GpuFamily,CompareType=CMP_Regex,MatchString="Adreno \\(TM\\) [7-8][0-9][0-9]"),(SourceType=SRC_AndroidVersion, CompareType=CMP_Regex,MatchString="([0-9]+).*"),(SourceType=SRC_PreviousRegexMatch,CompareType=CMP_GreaterEqual,MatchString="10"),(SourceType=SRC_SM5Available,CompareType=CMP_Equal,MatchString="true")))
++MatchProfile=(Profile="Android_Adreno7xx_Vulkan_SM5",Match=((SourceType=SRC_GpuFamily,CompareType=CMP_Regex,MatchString="Adreno \\(TM\\) 7[0-9][0-9]"),(SourceType=SRC_AndroidVersion, CompareType=CMP_Regex,MatchString="([0-9]+).*"),(SourceType=SRC_PreviousRegexMatch,CompareType=CMP_GreaterEqual,MatchString="10"),(SourceType=SRC_SM5Available,CompareType=CMP_Equal,MatchString="true")))
++MatchProfile=(Profile="Android_Adreno8xx_Vulkan_SM5",Match=((SourceType=SRC_GpuFamily,CompareType=CMP_Regex,MatchString="Adreno \\(TM\\) 8[0-9][0-9]"),(SourceType=SRC_AndroidVersion, CompareType=CMP_Regex,MatchString="([0-9]+).*"),(SourceType=SRC_PreviousRegexMatch,CompareType=CMP_GreaterEqual,MatchString="10"),(SourceType=SRC_SM5Available,CompareType=CMP_Equal,MatchString="true")))
 ; disable Vulkan on Adreno 660 devices with driver version <= 525
 +MatchProfile=(Profile="Android_Adreno6xx",Match=((SourceType=SRC_GpuFamily,CompareType=CMP_Regex,MatchString="Adreno \\(TM\\) 660"),(SourceType=SRC_GLVersion, CompareType=CMP_Regex,MatchString="V@([0-9]+)"),(SourceType=SRC_PreviousRegexMatch,CompareType=CMP_LessEqual,MatchString="525")))
 ; Enable Vulkan on Adreno 6xx devices with Android >= 10 and driver version >= 400
@@ -947,7 +949,7 @@ BaseProfileName=Android_Mid
 DeviceType=Android
 BaseProfileName=Android
 +CVars=sg.ViewDistanceQuality=2
-+CVars=sg.AntiAliasingQuality=1
++CVars=sg.AntiAliasingQuality=3
 +CVars=sg.ShadowQuality=2
 +CVars=sg.GlobalIlluminationQuality=2
 +CVars=sg.ReflectionQuality=2
@@ -968,7 +970,7 @@ BaseProfileName=Android
 +CVars=r.Shadow.MaxNumSpotShadowCacheUpdatesPerFrame=1
 +CVars=r.Shadow.DistanceScale=1.0
 +CVars=r.Shadow.CSM.MaxCascades=2 
-+CVars=r.ShadowQuality=2
++CVars=r.ShadowQuality=3
 +CVars=r.Shadow.CSMShadowDistanceFadeoutMultiplier=2.5
 +CVars=r.SSS.Quality=0
 +CVars=r.SSS.Scale=0
@@ -984,7 +986,11 @@ DeviceType=Android
 BaseProfileName=Android
 +CVars=r.Lumen.HardwareRayTracing=False
 
-[Android_Adreno_Vulkan_SM5 DeviceProfile]
+[Android_Adreno7xx_Vulkan_SM5 DeviceProfile]
+DeviceType=Android
+BaseProfileName=Android_Vulkan_SM5
+
+[Android_Adreno8xx_Vulkan_SM5 DeviceProfile]
 DeviceType=Android
 BaseProfileName=Android_Vulkan_SM5
 
@@ -1045,6 +1051,16 @@ BaseProfileName=Android_Adreno8xx
 ; enable Vulkan
 +CVars=r.Android.DisableVulkanSupport=0
 
+[Android_Adreno8xx DeviceProfile]
+DeviceType=Android
+BaseProfileName=Android_High
+
+[Android_Adreno8xx_Vulkan DeviceProfile]
+DeviceType=Android
+BaseProfileName=Android_Adreno8xx
+; enable Vulkan
++CVars=r.Android.DisableVulkanSupport=0
+
 ;----------------
 ; Xclipse devices
 
diff --git a/Engine/Source/Runtime/Renderer/Private/PostProcess/PostProcessEyeAdaptation.cpp b/Engine/Source/Runtime/Renderer/Private/PostProcess/PostProcessEyeAdaptation.cpp
index e9dab8b3f56d..95a2c0bb799a 100644
--- a/Engine/Source/Runtime/Renderer/Private/PostProcess/PostProcessEyeAdaptation.cpp
+++ b/Engine/Source/Runtime/Renderer/Private/PostProcess/PostProcessEyeAdaptation.cpp
@@ -642,7 +642,7 @@ class FSetupExposureIlluminanceCS : public FGlobalShader
 
 	static bool ShouldCompilePermutation(const FGlobalShaderPermutationParameters& Parameters)
 	{
-		return IsFeatureLevelSupported(Parameters.Platform, ERHIFeatureLevel::SM5);
+		return true;
 	}
 };
 
